docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -456,9 +456,29 @@ In the NHANES example, the full first sentence of the above quote was:
 3.2.1 Global Burden of Disease (GBD)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
-  Describe the best methodology/terminology for working with the "sex" variable from GBD
+The GBD study is very complex and many different data sources. Here, 
+we provide a general overview of how the demographics team works with the "sex" 
```

Comment:
The quotes help the reader understand that we are taking this name with a grain of salt, but I'd like to follow the advice we have elsewhere in the guide of only calling this "sex" once, and using more accurate terminology the rest of the time.